src/providers/fault-summary/cat-a-mod1/__tests__/fault-summary.cat-a-mod1.spec.ts

```diff
@@ -1,129 +1,124 @@
-  //     catAM1TestDataStateObject.emergencyStop.outcome = 'DF';
-  //     expect(FaultSummaryCatAM1Helper.hasEmergencyStopRidingFault(catAM1TestDataStateObject)).toBe(true);
-  //   });
-  //   it('should return false if a riding fault does not exist', () => {
-  //     catAM1TestDataStateObject.emergencyStop.outcome = 'S';
-  //     expect(FaultSummaryCatAM1Helper.hasEmergencyStopRidingFault(catAM1TestDataStateObject)).toBe(false);
-  //   });
-  // });
-
-  // describe('hasEmergencyStopSeriousFault', () => {
-  //   it('should return true if a serious fault exists', () => {
-  //     catAM1TestDataStateObject.emergencyStop.outcome = 'S';
-  //     expect(FaultSummaryCatAM1Helper.hasEmergencyStopSeriousFault(catAM1TestDataStateObject)).toBe(true);
-  //   });
-  //   it('should return false if a serious fault does not exist', () => {
-  //     catAM1TestDataStateObject.emergencyStop.outcome = 'DF';
-  //     expect(FaultSummaryCatAM1Helper.hasEmergencyStopSeriousFault(catAM1TestDataStateObject)).toBe(false);
-  //   });
-  // });
-
-  // describe('hasEmergencyStopDangerousFault', () => {
-  //   it('should return true if a dangerous fault exists', () => {
-  //     catAM1TestDataStateObject.emergencyStop.outcome = 'D';
-  //     expect(FaultSummaryCatAM1Helper.hasEmergencyStopDangerousFault(catAM1TestDataStateObject)).toBe(true);
-  //   });
-  //   it('should return false if a dangerous fault does not exist', () => {
-  //     catAM1TestDataStateObject.emergencyStop.outcome = 'DF';
-  //     expect(FaultSummaryCatAM1Helper.hasEmergencyStopDangerousFault(catAM1TestDataStateObject)).toBe(false);
-  //   });
-  // });
-
-  // describe('hasAvoidanceRidingFault', () => {
-  //   it('should return true if a riding fault exists', () => {
-  //     catAM1TestDataStateObject.avoidance.outcome = 'DF';
-  //     expect(FaultSummaryCatAM1Helper.hasAvoidanceRidingFault(catAM1TestDataStateObject)).toBe(true);
-  //   });
-  //   it('should return false if a riding fault does not exist', () => {
-  //     catAM1TestDataStateObject.avoidance.outcome = 'S';
-  //     expect(FaultSummaryCatAM1Helper.hasAvoidanceRidingFault(catAM1TestDataStateObject)).toBe(false);
-  //   });
-  // });
-
-  // describe('hasAvoidanceSeriousFault', () => {
-  //   it('should return true if a serious fault exists', () => {
-  //     catAM1TestDataStateObject.avoidance.outcome = 'S';
-  //     expect(FaultSummaryCatAM1Helper.hasAvoidanceSeriousFault(catAM1TestDataStateObject)).toBe(true);
-  //   });
-  //   it('should return false if a serious fault does not exist', () => {
-  //     catAM1TestDataStateObject.avoidance.outcome = 'DF';
-  //     expect(FaultSummaryCatAM1Helper.hasAvoidanceSeriousFault(catAM1TestDataStateObject)).toBe(false);
-  //   });
-  // });
+    it('should return true if a riding fault exists', () => {
+      catAM1TestDataStateObject.emergencyStop.outcome = 'DF';
+      expect(FaultSummaryCatAM1Helper.getEmergencyStopFaults(
```

Comment:
doing a .equals and comparing the returned object would be a better test here instead of checking the length